modules/oai-ran-du-k8s/main.tf

```
@@ -0,0 +1,75 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+resource "juju_model" "oai-ran-du" {
+  count = var.create_model == true ? 1 : 0
+  name  = var.model_name
```

```suggestion
  name  = var.model
```

Don't use this suggestion as-is, but the variable will be named `model` in the future, as this will eventually change to a UUID.

---

Are you sure that's correct? Isn't it the references that will change? 
This variable is just passing the name of the model to create. I guess it's fine to call it `model_name`. 
What I would expect to change is how we use a reference to the created object, i.e.:
```
resource "juju_application" "cu" {
  name  = var.app_name
  model = var.model_name  <-- This should be accepting the object instead of a string
  (...)
}
```